lib/src/model/accessor.dart

```diff
@@ -16,11 +16,27 @@ class Accessor extends ModelElement implements EnclosedElement {
 
   Accessor(PropertyAccessorElement element, Library library,
       PackageGraph packageGraph,
-      [Member /*?*/ originalMember])
+      [ExecutableMember /*?*/ originalMember])
       : super(element, library, packageGraph, originalMember);
 
   @override
-  CallableElementTypeMixin get modelType => super.modelType;
+  PropertyAccessorElement get element => super.element;
+
+  @override
+  ExecutableMember get originalMember => super.originalMember;
+
+  CallableElementTypeMixin _modelType;
+  CallableElementTypeMixin get modelType {
+    if (_modelType == null) {
+      if (originalMember != null) {
+        _modelType =
+            ElementType.from(originalMember.type, library, packageGraph);
```

Comment:
Would `originalMember ?? element` work here?

_modelType =
            ElementType.from((originalMember ?? element).type, library, packageGraph);

Reply:
yes, this can all be collapsed.  Done here and elsewhere